hibernate-reactive-core/src/test/java/org/hibernate/reactive/CompositeIdTest.java

```diff
@@ -50,7 +51,8 @@ public void after(TestContext context) {
 
 	private CompletionStage<String> selectNameFromId(Integer id) {
 		return connection().thenCompose( connection -> connection.select(
-				"SELECT name FROM Pig WHERE id = $1", new Object[]{id} ).thenApply(
+				DatabaseConfiguration.statement("SELECT name FROM Pig WHERE id = ",""), 
```

Comment:
Actually it would be better if we simply got rid of all these hand-coded SQL statements from the test suite. They're an absolute *pain* to maintain.

Reply:
But we can do it in a separate PR

Reply:
Well, whatever is easiest. I only mention it here because getting rid of them would eliminate the need for the `DatabaseConfiguration.statement()` method.

Reply:
Lets do it in a separate PR, but how do you intend to eliminate hand-coded SQL? I think it is valuable to manually verify the contents of the DB with something besides HR to prevent garbage-in-garbage-out mistakes.